docs/team/zacwong2151.md

```diff
@@ -3,44 +3,48 @@ layout: default.md
 
-* **Enhancements to existing features**:
-    * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-    * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+* **Project management**:
+    * Help set up the issue tracker and milestone management on GitHub
 
 * **Documentation**:
     * User Guide:
-        * Added documentation for the features `delete` and `find` [\#72]()
-        * Did cosmetic tweaks to existing documentation of features `clear`, `exit`: [\#74]()
+        * Added documentation for the features `efind`: [\#207](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/207) and `vfind`: [\#103](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/103)
+        * Added documentation for the features `undo` and `redo`: [\#206](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/206)
     * Developer Guide:
-        * Added implementation details of the `delete` feature.
+        * Added use cases for event delete feature: [\#67](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/67)
+        * Added implementation details for undo and redo features, inclusive of UML class diagrams: [#216](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/216)
+        * Added a Planned Enhancements section to propose fixes to add in the future: [\#215](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/215)
+        * Added a Instructions for Manual Testing section to give testers guidance on using our app: [\#218](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/218)
 
 * **Community**:
-    * PRs reviewed (with non-trivial review comments): [\#12](), [\#32](), [\#19](), [\#42]()
-    * Contributed to forum discussions (examples: [1](), [2](), [3](), [4]())
-    * Reported bugs and suggestions for other teams in the class (examples: [1](), [2](), [3]())
-    * Some parts of the history feature I added was adopted by several other class mates ([1](), [2]())
-
-* **Tools**:
-    * Integrated a third party library (Natty) to the project ([\#42]())
-    * Integrated a new Github plugin (CircleCI) to the team repo
-
-* _{you can add/remove categories in the list above}_
+    * PRs reviewed (with non-trivial review comments): [\#122](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/122), [\#69](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/69), [\#113](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/113), [\#82](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/82)
```

Comment:
can add your `PE-D` reviews you did as well cos you reported 9 bugs (achievement!) - probably can list 2 or 3 of them

Reply:
Eh PE-D is not related to our team, can include meh

Reply:
ya that's the whole point for the Community section in PPP

Reply:
ohh i didnt know u can do that thanks